system/lib/compiler-rt/lib/ubsan_minimal/readme.txt

```diff
@@ -0,0 +1,14 @@
+These files are from compiler-rt.
+
+Last Changed Rev: 351636
+Last Changed Date: 2019-01-19
+
+===========================================================================
+
+* `ubsan_minimal_handlers.cpp` -- changed to use
+  `emscripten_wasm_return_address` as `__builtin_return_address` is not yet
+  available in clang target `wasm-unknown-emscripten`.
+* `sanitizer_atomic.h` -- based on `sanitizer_atomic.h`,
+  `sanitizer_atomic_clang.h` and `sanitizer_atomic_clang_other.h` in
+  upstream, combined into one to avoid pulling in and porting the entire
+  `sanitizer_common` directory from upstream.
```

Comment:
`sanitizer_atomic.h` here is mostly a stripped down version of the header in `compiler-rt` to make `ubsan_minimal_handlers.cpp` compile without porting over all its dependencies. It will be removed once we port `sanitizer_common`.

Reply:
I'm tempted to add this to  `lib/compiler-rt/readme.txt` since its kind of part of compiler-rt isn't it?  Maybe you want to be able to rev the two independently?